packages/teleport/src/Player/Player.tsx

```diff
@@ -16,17 +16,26 @@ limitations under the License.
 
 import React from 'react';
 import styled from 'styled-components';
-import { useParams } from 'teleport/components/Router';
+import { useParams, useLocation } from 'teleport/components/Router';
 import { Flex } from 'design';
 import Tabs, { TabItem } from './PlayerTabs';
 import SshPlayer from './SshPlayer';
+import { DesktopPlayer } from './DesktopPlayer';
 import ActionBar from './ActionBar';
 import session from 'teleport/services/session';
 import { colors } from 'teleport/Console/colors';
 import { UrlPlayerParams } from 'teleport/config';
+import { getUrlParameter } from 'teleport/services/history';
 
 export default function Player() {
   const { sid, clusterId } = useParams<UrlPlayerParams>();
+  const recordingType = getUrlParameter('recordingType', useLocation().search);
+  if (recordingType !== 'ssh' && recordingType !== 'desktop') {
+    throw new Error(
+      `Invalid recording type: ${recordingType}, should be 'ssh' or 'desktop'`
+    );
+  }
```

Comment:
i'm not sure if we should be throwing... i rarely see this, wouldn't it be better if we show an error banner like we do everywhere else?

Reply:
https://gravitational.slack.com/archives/C02DQ1C2BMW/p1643753870653229